v2/protocol/result.go

```diff
@@ -47,10 +48,19 @@ func IsACK(target Result) bool {
 	return ResultIs(target, ResultACK)
 }
 
+// IsNACK true means the recipient did not acknowledge the event.
 func IsNACK(target Result) bool {
 	return ResultIs(target, ResultNACK)
 }
 
+// Undelivered true means the target result is not an ACK/NACK, but some other
+// error unrelated to delivery not from the intended recipient. Likely target
+// is an error that represents some part of the protocol is misconfigured or
+// the event that was attempting to be sent was invalid.
+func Undelivered(target Result) bool {
```

Comment:
The naming is not consistent with `IsACK` and `IsNACK`, perhaps it should be `IsUndelivered`?

Reply:
Sure, Grouping these together seems fine.